packages/create-remix/package.json

```diff
@@ -17,7 +17,28 @@
     "create-remix": "dist/cli.js"
   },
   "dependencies": {
-    "@remix-run/dev": "1.19.0"
+    "arg": "^5.0.1",
+    "chalk": "^4.1.2",
+    "execa": "5.1.1",
+    "gunzip-maybe": "^1.4.2",
+    "log-update": "^5.0.1",
+    "node-fetch": "^2.6.9",
+    "proxy-agent": "^5.0.0",
```

Comment:
Can we update this to `6.3.0` while we're making this overhaul to resolve https://github.com/remix-run/remix/pull/6862?  I think `proxy-agent` can also be removed from remix/dev package.json since (I think) it was only used in `cli/create.ts` which is now deleted.